code/modules/flufftext/Hallucination.dm

```diff
@@ -471,10 +471,13 @@ GLOBAL_LIST_INIT(hallucinations_major, list(
 	var/image/A = null
 	var/kind = force_kind ? force_kind : pick("monkey","corgi","carp","skeleton","demon","zombie")
 	feedback_details += "Type: [kind]"
+	var/list/nearby
+	if(skip_nearby)
+		nearby = get_hearers_in_view(7, target)
```

Comment:
why did you move this ?

Reply:
run a laggy proc 1 time instead of for every mob in glob.moblist @optimumtact

Reply:
oh right, that makes sense